rbac/migration_tool/utils.py

```diff
@@ -74,9 +49,8 @@ def write_relationships(relationships):
         try:
             stub.CreateTuples(request)
         except grpc.RpcError as err:
-            error = GRPCError(err.value)
```

Comment:
Will the GRPCError(err) work? We got the GRPCError from QE, thought it might be useful to construct the error

Reply:
as I remember I think that in `error.reason` is more details - all errors are not is in err.details() -  @vbelchio  is it correct ?

Reply:
I'll pull this commit out for now. I was getting errors here when the connection wasn't successful, but it's outside the scope of this anyway and it sounds like I'm missing some context. :)